lib/widgets/action_sheet.dart

```diff
@@ -71,22 +71,43 @@ class ShareButton extends MessageActionSheetMenuItemButton {
 
   @override get onPressed => (BuildContext context) async {
     // Close the message action sheet; we're about to show the share
-    // sheet. (We could do this after the sharing Future settles, but
-    // on iOS I get impatient with how slowly our action sheet
-    // dismisses in that case.)
+    // sheet. (We could do this after the sharing Future settles
+    // with [ShareResultStatus.success], but on iOS I get impatient with
+    // how slowly our action sheet dismisses in that case.)
     // TODO(#24): Fix iOS bug where this call causes the keyboard to
     //   reopen (if it was open at the time of this
     //   `showMessageActionSheet` call) and cover a large part of the
     //   share sheet.
     Navigator.of(context).pop();
 
+    final rawContent = await fetchRawContentWithFeedback(
+      context: messageListContext,
+      messageId: message.id,
+      errorDialogTitle: 'Sharing failed',
```

Comment:
Rebasing on main, the similar call sites now look like this:
```dart
      errorDialogTitle: zulipLocalizations.errorQuotationFailed,
```

So this is probably a good time for you to give our spiffy new i18n workflow a try.

Reply:
Oh nice!